codegen/service_test.go

```diff
@@ -30,7 +30,7 @@ import (
 	"github.com/uber/zanzibar/codegen"
 )
 
-var updateGoldenFile = flag.Bool("update", false, "Updates the golden files with expected response.")
+var updateGoldenFile = flag.Bool("update", true, "Updates the golden files with expected response.")
```

Comment:
You can do `go test ./codegen/ -update` ;)

We want this flag to stay defaulted to false.